frontend/transformation-playground/.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "git.ignoreLimitWarning": true
```

Comment:
Can we remove this or do we need to add additional .gitignore exclusions to prevent this from coming up?

Reply:
Ah, this is likely for a local issue. I'll add the `.vscode/settings.json` to `.gitignore`

Reply:
Looks like this file is still in the tree, I believe its automatically created when you click 'don't ask me again' when you get the warning from git about too many files being in the project - likely due to a missing .gitignore entry targeting the node_module directory.

I don't think we need to ,gitignore these files as for example they are used with VSCode's python's plugin to isolate the venv.   I think this setting `git.ignoreLimitWarning` isn't one we'd want to include.  What do you think about removing the git ignore and deleting this file?

Reply:
I fear that in that case it'll be reintroduced with every commit I do because it's being recreated automatically. Let me play with moving this exclusion to my global gitignore instead.